### ISSUE TITLE
[BUG] TypeError: path.dirname is not a function

### ISSUE THREAD

**Issue by ClintH** [lukasbach/monaco-editor-auto-typings#25]
I'm also having the error as #9, 'path.dirname is not a function' error, even though importing from 'monaco-editor-auto-typings/custom-editor' and passing in a monaco instance.

I am using Lit, TS & Vite and the latest version of monaco-editor-auto-typings.

```
Uncaught (in promise) TypeError: path.dirname is not a function
at AutoTypingsCore2. (chunk-D6HROQ3V.js?v=c1a822df:1459:94)
at step (chunk-D6HROQ3V.js?v=c1a822df:1308:23)
at Object.next (chunk-D6HROQ3V.js?v=c1a822df:1255:18)
at chunk-D6HROQ3V.js?v=c1a822df:1241:71
at new Promise ()
at __awaiter (chunk-D6HROQ3V.js?v=c1a822df:1223:14)
at AutoTypingsCore2.resolveContents (chunk-D6HROQ3V.js?v=c1a822df:1442:16)
at new AutoTypingsCore2 (chunk-D6HROQ3V.js?v=c1a822df:1353:14)
at Function. (chunk-D6HROQ3V.js?v=c1a822df:1370:24)
at step (chunk-D6HROQ3V.js?v=c1a822df:1308:23)
```

[Here's a minimal repro](https://stackblitz.com/edit/vitejs-vite-ului2l?file=src/code-editor.ts)

I am not savvy when it comes to build processes, so maybe there is a Vite/Rollup setting I'm missing.

Any tips appreciated! :)

**Comment by burtek** [comment]
Honestly, not seeing that issue happen in the given repro. It seems to be working fine for me.

**Comment by ClintH** [comment]
Thanks for taking a look! You don't see the error in the DevTools console? I see it using Edge & Chrome browsers.

**Comment by ClintH** [comment]
![Screen Shot 2022-11-08 at 17 14 33](https://user-images.githubusercontent.com/533348/200622175-af553d1c-4404-4c12-9a3f-ae2cc243c39d.jpg)

**Comment by burtek** [comment]
Yeah, see it now. Had some wrong filters set in devtools console 🤦

**Comment by miraries** [comment]
Just ran into this, I'm using vite as well (with svelte kit).
This works fine in the demo due to `path-browserify`, so fixed it by adding a similar polyfill for the `path` module using an esbuild plugin from [this gist](https://gist.github.com/FbN/0e651105937c8000f10fefdf9ec9af3d).

**Comment by valerii15298** [comment]
Same problem for me :(
Will be support for vite added???
Would be very much helpful...

**Comment by valerii15298** [comment]
@miraries can you please share what exactly and how did you use from that gist? I am looking into it but there are so much stuff that I cannot understand what I should take from it...

**Comment by miraries** [comment]
@valerii15298 Do `npm i @esbuild-plugins/node-modules-polyfill`, then add the following `resolve` config in your `vite.config.js` :
```javascript
export default defineConfig({
  resolve: {
    alias: {
      path: 'rollup-plugin-node-polyfills/polyfills/path'
    }
  }
});
```

this just adds a polyfill for the `path` module.

**Comment by valerii15298** [comment]
@miraries thank you very much!!!
